### PR TITLE
Add 'hidden' option to parameter object

### PR DESCRIPTION
**Pull request by aweiteka**
Not sure if we can just merge this in without bumping the version. If not maybe we need to create a "next" branch that we can cherry-pick commits from.

### CODE REVIEW

**Comment by vpavlin** [comment]
IMO it's basically a bug fix so I'd say we could add it to 0.0.1 and I'd add it also to atomicapp if you file an issue once this is merged so LGTM from me.

File: spec/0.0.2/README.md [hunk 1]
```diff
@@ -184,12 +184,14 @@ Field Name | Type | Description
 <a name="parametersDescription"></a>description | `string` | **Required.** A human readable description of the parameter.
 <a name="parametersConstraints"></a>constraints | [ConstraintObject](#constraintObject) | **Optional** An optional definition of constraints to the parameter.
 <a name="parametersDefault"></a>default | `string` | **Optional** An optional default value for the parameter.
+<a name="parametersHidden"></a>hidden | `string` | **Optional** An optional boolean signifying the parameter should be obscured when displayed.
```
Comment (goern) [inline spec/0.0.2/README.md:187]: type should be boolean and not string

**Comment by goern** [comment]
looks like a bugfix of 0.0.2 to me, so we should merge it.

**Comment by vpavlin** [comment]
LGTM

**Pull request merged**